src/main/java/org/folio/tlib/postgres/PgCqlQuery.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.tlib.postgres;
+
+import org.folio.tlib.postgres.impl.PgCqlQueryImpl;
+
+public interface PgCqlQuery {
```

Comment:
Because PgCqlQuery is a much more complicated implementation. I prefer using an interface and static constructor, even if there's only one implementation.

Reply:
Fine.